roles/openshift_node/meta/main.yml

```diff
@@ -12,5 +12,6 @@ galaxy_info:
   categories:
   - cloud
 dependencies:
+- role: openshift_clock
```

Comment:
I expect this will make it in first and I can fix things up in my rebase. Already a few rebases behind anyway.